### DISCUSSION TITLE
installation question on windows

### DISCUSSION THREAD

**Discussion by b4wen** [NanoComp/meep#2833] [Q&A]
Hi, 
I have been struggling installing MEEP for a few days. If anyone can help me with this it would be very appreciated! 
My first question:
Did mini coda website moved? When I run it, it gives me 404 error. 
![image](https://github.com/NanoComp/meep/assets/48703883/d491b00b-9f57-426b-9d1c-3f1ae9f3ee57)
My second question:
Then I tried to use Ubuntu to install meep, it didn't give any error until I try to call any attribute from Meep:
![image](https://github.com/NanoComp/meep/assets/48703883/56aeeec5-73ba-4f09-b707-5e8e19652298)
I'm very confused now. What should I do to install Meep? 
Thanks to any response in advance.

**Comment by stevengj** [comment]
I don't think there is an up-to-date `pip` package for meep — note that the latest Meep release is 1.28 so 1.0.6 is extremely old.

You could try the Ubuntu package, `sudo apt-get install python3-meep`, though I don't know if that's been updated recently.

For Anaconda, see [the instructions here](https://docs.anaconda.com/free/miniconda/miniconda-install/).
